src/mjolnir/pbfgraphparser.cc

```diff
@@ -30,7 +30,7 @@ using namespace valhalla::mjolnir;
 namespace {
 
 // Will throw an error if this is exceeded. Then we can increase.
```

Comment:
Just a minor nitpick, and probably not really important, but this comment is not completely true. It doesn't throw an error but a warning, as it resizes the bitsize to deal with the warning and nothing bad happens, if I understood the warning and [the code](https://github.com/valhalla/valhalla/blob/9551ddeeb27f538d0670d553418d5c441bfbada8/valhalla/mjolnir/idtable.h#L76) correctly. I've been getting that warning for the last 4 or 5 tile builds, but gave it not much importance because it seemed to be dealt with. Now I see that I missed the hardcoded constant here.

Reply:
@Rui-Santos you are correct i remember making this change a few months ago!